### ISSUE TITLE
Custom attribute(Customer Address type) not saved when Create New Order from admin

### ISSUE THREAD

**Issue by ronakChauhanKrish** [magento/magento2#10805]
### Preconditions

1. Magento 2.1.7
2. Identified on PHP 7.0.9-1
 

### Steps to reproduce

1. Create Customer address attribute using installdata script as below:
`$customerSetup = $this->customerSetupFactory->create(['setup' => $setup]);

        $customerSetup->addAttribute('customer_address', 'suburb', [
            'label' => 'Suburb',
            'input' => 'text',
            'type' => 'varchar',
            'source' => '',
            'required' => true,
            'position' => 70,
            'visible' => true,
            'system' => false,
            'is_used_in_grid' => false,
            'is_visible_in_grid' => false,
            'is_filterable_in_grid' => false,
            'is_searchable_in_grid' => false,
            'backend' => ''
        ]);

        
        $attribute = $customerSetup->getEavConfig()->getAttribute('customer_address', 'suburb')
            ->addData(['used_in_forms' => [
                'adminhtml_customer_address',
                'customer_address_edit',
                'customer_register_address'
            ]]);
        $attribute->save();`
2. Open Admin >> Sales >> Order >> Create New Order
3. Select Store >> Address Information >> Fill address with our Custom attribute(**Suburb**)
4. Select Payment Method and Shipping Method
5. Submit Order

### Expected result

1. Suburb should be saved same as other customer address attributes
2. As we have already filled value in Custom attribute field(**Suburb**) It should not generate any required field value PHP error.
3. Even if We have already save value in Customer >> Address then It must be visible on Create new Order >> Billing Address/ Shipping Address form

### Actual result

1. 
![new order orders operations sales magento admin](https://user-images.githubusercontent.com/28439360/30157100-fcfdc124-93de-11e7-8afb-fa3c3be741e7.png)
2. 
![new order orders operations sales magento admin 1](https://user-images.githubusercontent.com/28439360/30157140-18ce0846-93df-11e7-9ff2-cb44cb646274.png)

3. Value not saved while other attributes saved and retrieved as well.

**Comment by marcinfr** [comment]
im on it

**Comment by marcinfr** [comment]
It is not saved, because order address values are stored in "quote_address" and "sales_order_address" tables. By creating new customer address attribute, columns are not automatically created in the above tables. 
There are more steps to add new address attribute in correct way:
1. create customer address attribute
2. create new columns in quote_address and sales_order_address:
``` 
$salesSetup = $this->salesSetupFactory->create(['setup' => $setup]);
$salesSetup->addAttribute('order_address', 'suburb', ['type' => 'text']);
$quoteSetup = $this->quoteSetupFactory->create(['setup' => $setup]);
$quoteSetup->addAttribute('quote_address', 'suburb', ['type' => 'text']);
```
3. add fieldset in xml file to copy new field value from quote address to order address
``` 
<fieldset id="sales_convert_quote_address">
     <field name="suburb">
          <aspect name="to_order_address" />
     </field>
</fieldset>
```
4. create new setter method in order address class:
```
public function setSuburb($suburb)
{
     return $this->setData('suburb', $suburb);
}
```

**Comment by magento-engcom-team** [comment]
@ronakChauhanKrish thank you for your bug report.
We've created internal ticket MAGETWO-75304 to track progress on the issue

**Comment by mallafreDG** [comment]
I am working on it at #mm17es

**Comment by mallafreDG** [comment]
Reproduced the same error in Magento 2.2.1 with PHP 7.0.25

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [x] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [x] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by magento-engcom-team** [comment]
@engcom-backlog-nazar Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-97682`, `MAGETWO-97683` were created

**Comment by DmitryFurs** [comment]
Yes the ticket is still relevant.

**Comment by yohanespradono** [comment]
is this fixed on 2.4.6? apparently i am still getting this issue.
